cmd/bucket-replication.go

```diff
@@ -1582,6 +1583,7 @@ func NewReplicationPool(ctx context.Context, o ObjectLayer, opts replicationPool
 		mrfWorkerKillCh: make(chan struct{}, failedWorkers),
 		resyncer:        newresyncer(),
 		mrfSaveCh:       make(chan MRFReplicateEntry, 100000),
+		mrfStopCh:       make(chan struct{}, 1),
```

Comment:
Suggestion: We could use an unbuffered channel to signal mrf routine that we're done by closing it.
```suggestion
		mrfStopCh:       make(chan struct{}),
```